table.cc

```diff
@@ -412,6 +412,26 @@ void table::add_maintenance_sstable(sstables::shared_sstable sst) {
     refresh_compound_sstable_set();
 }
 
+void table::do_update_off_strategy_trigger() {
+    _off_strategy_trigger.rearm(timer<>::clock::now() +  std::chrono::minutes(5));
+}
+
+// If there are more sstables to be added to the off-strategy sstable set, call
+// update_off_strategy_trigger to update the timer and delay to trigger
+// off-strategy compaction. The off-strategy compaction will be triggered when
+// the timer is expired.
+void table::update_off_strategy_trigger() {
```

Comment:
perhaps reset rather than update?

Reply:
I think update is more accurate than reset because we are not resetting the timer to an initial state but keep updating it.